src/nvapi/nvapi_d3d_low_latency_device.cpp

```diff
@@ -110,6 +110,31 @@ namespace dxvk {
             GetFrameIdGenerator(d3dLowLatencyDevice.ptr())->GetLowLatencyDeviceFrameId(frameID), markerType));
     }
 
+    std::optional<uint32_t> NvapiD3dLowLatencyDevice::ToMarkerType(NV_LATENCY_MARKER_TYPE markerType) {
+        switch (markerType) {
+            case (SIMULATION_START):
+            case (SIMULATION_END):
+            case (RENDERSUBMIT_START):
+            case (RENDERSUBMIT_END):
+            case (PRESENT_START):
+            case (PRESENT_END):
+            case (INPUT_SAMPLE):
+            case (TRIGGER_FLASH):
+                return markerType;
+            // VkLatencyMarkerNV misses PC_LATENCY_PING and all following enum values are offset
+            // See https://registry.khronos.org/vulkan/specs/1.3-extensions/html/vkspec.html#VUID-VkSetLatencyMarkerInfoNV-marker-parameter
+            case (PC_LATENCY_PING):
+                return {};
+            case (OUT_OF_BAND_RENDERSUBMIT_START):
+            case (OUT_OF_BAND_RENDERSUBMIT_END):
+            case (OUT_OF_BAND_PRESENT_START):
+            case (OUT_OF_BAND_PRESENT_END):
+                return markerType - 1;
+        }
+
+        return {};
```

Comment:
One concern about throwing `PC_LATENCY_PING` and invalid values in the same bag is that if new values are introduced in the future, detecting that they are used will be much harder. So, I'd prefer just logging invalid values here when one is encountered (at least once when we are passed anything outside known `SIMULATION_START` .. `OUT_OF_BAND_PRESENT_END` range), but it's not impossible that this won't ever matter.

Reply:
Good idea, I've added the log statement, also unconditionally because this should be in our faces.